src/server/cms/infra/repositories/strapi.mapper.ts

```diff
@@ -312,9 +312,10 @@ export function mapVideoCampagneApprentissage(video: Strapi.CollectionType.Video
 
 export function mapFormationInitiale(formationInitiale: Strapi.CollectionType.FormationInitialeDetail): FormationInitialeDetailCMS {
 	return {
-		attendusParcoursup: formationInitiale?.attendusParcoursup,
-		conditionsAcces: formationInitiale?.conditionsAcces,
-		description: formationInitiale?.description,
-		poursuiteEtudes: formationInitiale?.poursuiteEtudes,
+		attendusParcoursup: formationInitiale.attendusParcoursup,
+		conditionsAcces: formationInitiale.conditionsAcces,
+		description: formationInitiale.description,
+		poursuiteEtudes: formationInitiale.poursuiteEtudes,
+		updatedAt: formationInitiale.updatedAt,
```

Comment:
nitpick: Pour ne pas avoir dans l'anglais dans notre domain 😬
```suggestion
		dateDeMiseAJour: formationInitiale.updatedAt,
```